src/pip/_internal/cli/req_command.py

```diff
@@ -119,12 +119,17 @@ def _build_session(
         else:
             ssl_context = None
 
+        if "parallel_downloads" in options.__dict__:
+            parallel_downloads = options.parallel_downloads
+        else:
+            parallel_downloads = 1
```

Comment:
Does this branch ever get used? I thought `default` would cover the case where the user does not supply a value.

Reply:
the `parallel_downloads` option is only added to the `install` and `download` sub-commands, so in other sub-commands (index, list etc.) there is no `options.parallel_downloads`.  So in those cases I'm setting `parallel_downloads` to 1.